src/main/java/at/hannibal2/skyhanni/features/garden/visitor/VisitorAPI.kt

```diff
@@ -131,6 +131,8 @@ object VisitorAPI {
     ) {
         var offersAccepted: Int? = null
         var pricePerCopper: Int? = null
+        var totalPrice: Double? = null
+        var totalReward: Double? = null
```

Comment:
This does only use copper price, not the reward item price.

Reply:
From my understanding this part of t
```kotlin
val price = internalName.getPrice() * amount
if (readingShoppingList) {
    totalPrice += price
    lastFullPrice += price
} else {
    foundRewards.add(internalName)
    totalPrice -= price
}
```